### PR TITLE
new contact page

### PR DESCRIPTION
**Pull request by keithbradley1**
new contact page

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <li><a href="glennballs@facebook.com" title="Facebook" class="">Facebook</a></li>
+      </ul>
+      <ul>
+        <li>Russell Balls - Chief Technical Process Supervisor of Intergalactic Manipulation and Sith Lordery and Ball Shaped Destructive Devices</li>
+        <li><a href="tel:1-408-555-5555">1-408-555-5555</a></li>
+        <li><a href="mailto:nowhere@mozilla.org">email</li>
+        <li><a href="twitter.russellballs@twitter.com" title="Twitter" class="">Twitter</a></li>
+        <li><a href="russellballs@facebook.com" title="Facebook" class="">Facebook</a></li>
+      </ul><ul>
+        <li>Keith Balls - First Imperial Executive to the Golabal Comitee for the Advancement of Balls</li>
```
Comment (RussellRiesJr) [inline contact.html]: committee, not comitee

**Comment by RussellRiesJr** [comment]
:+1:

File: contact.html [hunk 2]
```diff
@@ -19,7 +19,7 @@
   </header><!-- /header -->
 	<article class="body">
     <h1>Contact Balls</h1>
```
Comment (glenngineer1) [inline contact.html:21]: indent the h1

**Comment by glenngineer1** [comment]
See one comment

**Comment by glenngineer1** [comment]
:+1:

**Comment by glenngineer1** [comment]
:+1:

**Pull request merged**